R/plot_aggregate_loanbooks.R

```diff
@@ -63,69 +63,85 @@ plot_aggregate_loanbooks <- function(config) {
   )
   col_select_company_aggregated_alignment <- c(by_group, names(col_types_company_aggregated_alignment[["cols"]]))
 
-  company_aggregated_alignment_net  <-
-    readr::read_csv(
-      file = file.path(
-        output_analysis_aggregated_dir,
-        glue::glue("company_exposure_net_aggregate_alignment{file_by_group}.csv")
-      ),
-      col_types = col_types_company_aggregated_alignment,
-      col_select = dplyr::all_of(col_select_company_aggregated_alignment)
-    )
+  if (file.exists(file.path(output_analysis_aggregated_dir, glue::glue("company_exposure_net_aggregate_alignment{file_by_group}.csv")))) {
+    company_aggregated_alignment_net  <-
+      readr::read_csv(
+        file = file.path(
+          output_analysis_aggregated_dir,
+          glue::glue("company_exposure_net_aggregate_alignment{file_by_group}.csv")
+        ),
+        col_types = col_types_company_aggregated_alignment,
+        col_select = dplyr::all_of(col_select_company_aggregated_alignment)
+      )
```

Comment:
I'd rather see this filename "glued" together once above, then tested for existence, and then possibly read from.

Reply:
done